crates/sui-source-validation-service/src/lib.rs

```diff
@@ -73,10 +80,14 @@ pub async fn verify_package(
     for v in &compiled_package.package.root_compiled_units {
         match v.unit {
             CompiledUnitEnum::Module(ref m) => {
-                map.insert((address, m.name), v.source_path.to_path_buf())
+                let path = v.source_path.to_path_buf();
```

Comment:
Does this path point into the `sources/` directory or into the `build/<package>/sources/` directory?  I think the latter is going to be preferable because the source language supports multiple modules existing in a single file, and the compiler chunks them up, each into their own file, with a name matching their module name (in the latter directory), so we can take advantage of it having done that work.

Reply:
TIL! Nope this doesn't point to the `build` path. 

Your comment got me excited--I knew about multiple modules in the same file and wondered how we'd deal with that. After looking into, the `build` dir doesn't quite chunk up files per module (I was very curious how it would handle comments between modules, for example, if it did that). Unfortunately it appears to just duplicate the file source, so something like:

```
module a::x {}

module a::y {}
```

means it will create two files, `x.move` and `y.move` with the same contents. Still! This, using the `build` path is better than the current `sources` path, which would not resolve module `y` to any file (and by using `build` we would discover `y.move`).

Probably worth figuring out how to actually chunk modules. But, I have some thoughts on doing client-side / Explorer operations that might work out better. For example, I'd like to be able to jump to function headers that align the `fun` declaration for both bytecode and source code (we might just look for a common header, like `diff` does, say). If we can get usability like that, it would also work well enough IMO to just have the web view jump to the module declaration, even though we render a single file.

I'll follow up to resolve to the `build` path!

Reply:
> After looking into, the build dir doesn't quite chunk up files per module (I was very curious how it would handle comments between modules, for example, if it did that). Unfortunately it appears to just duplicate the file source

Ah, apologies, wishful thinking on my part!!  Re: comments, I believe Move has a notion of (module) doc comments which should associate themselves with the module that follows them, but otherwise, yes, not sure about your regular `//` comments.